binding.cc

```diff
@@ -1015,6 +1025,86 @@ namespace zmq {
 
 #endif
 
+  NAN_METHOD(Socket::Readv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::Undefined());
+
+    int events;
+    size_t events_size = sizeof(events);
```

Comment:
`sizeof` is a keyword, not a function (but the parens are not always optional), so it should have a space after